docs/content/entity-templates.md

```diff
@@ -0,0 +1,58 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+------
+
+## Creating entity templates
+
+To create an [entity](https://docs.improbable.io/reference/latest/shared/glossary#entity) in the SpatialOS GDK for Unity, you need to create a template which specifies which components should be added to the entity and allows you to specify [authority](authority.md) on a per-[component](https://docs.improbable.io/reference/latest/shared/glossary#component) basis.
+
+The entity template can be created using the `EntityBuilder` which is part of the `Improbable.Gdk.Core` assembly. The `EntityBuilder` class has the following public methods:
```

Comment:
"The entity template can be created using the `EntityBuilder`..."
->
"Create the entity template using the `EntityBuilder`..."

(The original is in passive voice so I reader has to infer that it's they who have to do something. Our text doesn't explicitly tell them.)